### ISSUE TITLE
can it support IE 11 ??

### ISSUE THREAD

**Issue by kaykie** [joeldenning/coexisting-vue-microfrontends#27]

**Comment by joeldenning** [comment]
Yep. vue, single-spa, and systemjs all support IE11. I'd need more detail to understand what isn't working for you.

**Comment by kaykie** [comment]
when i open http://10.10.33.71:5000/ on IE this error happen
![image](https://user-images.githubusercontent.com/26200899/69046848-e54fa280-0a34-11ea-97c8-a75a0319180e.png)

then i add the babel-core
it can open index
when i route to app1 this error happen 

![image](https://user-images.githubusercontent.com/26200899/69046603-5a6ea800-0a34-11ea-9a6b-ca8f6a0f6840.png)

i didn't change you code 

i found the reason :
import { setPublicPath } from 'systemjs-webpack-interop'

setPublicPath('app2')

IE don't support this syntactic 
![image](https://user-images.githubusercontent.com/26200899/69047126-6f980680-0a35-11ea-934e-06a4a927768e.png)

but i can't solve it 
can you help me?
